ci/docker/runtime_functions.sh

```diff
@@ -1453,7 +1453,7 @@ nightly_test_imagenet_inference() {
 nightly_test_image_classification() {
     set -ex
     export DMLC_LOG_STACK_TRACE_DEPTH=10
-    ./tests/nightly/test_image_classification.sh
+    # TODO: the test script needs to be added (18549)
```

Comment:
Please do a proper cleanup instead of just removing references..

Reply:
Addressed.